src/EnergyPlus/FanCoilUnits.cc

```diff
@@ -1309,7 +1309,7 @@ namespace FanCoilUnits {
             !DataHVACGlobals::ZoneCompTurnFansOff) {
             Node(InletNode).MassFlowRate = FanCoil(FanCoilNum).MaxAirMassFlow;
             Node(InletNode).MassFlowRateMaxAvail = Node(InletNode).MassFlowRate;
-            Node(InletNode).MassFlowRateMinAvail = Node(InletNode).MassFlowRate;
+            Node(InletNode).MassFlowRateMinAvail = 0.0;
 
```

Comment:
Fan Coil Units supply air inlet node minimum available flow rate was set zero instead of design maximum flow rate. Setting supply air inlet node minimum available flow rate to design maximum flow forces the Fan:VariableVolume to run at design maximum flow rate regardless of the load.